src/azure-cli/azure/cli/command_modules/iot/tests/hybrid_2020_09_01/test_iot_commands.py

```diff
@@ -0,0 +1,550 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+
+from azure.cli.testsdk import ResourceGroupPreparer, ScenarioTest, StorageAccountPreparer
+from azure_devtools.scenario_tests import AllowLargeResponse
+from .recording_processors import KeyReplacer
+import unittest
+
+
+class IoTHubTest(ScenarioTest):
+
+    def __init__(self, method_name):
+        super(IoTHubTest, self).__init__(
+            method_name, recording_processors=[KeyReplacer()]
+        )
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(location='westus2')
+    @StorageAccountPreparer()
+    @unittest.skip('Need to try with passing SAS key in environment.')  # TODO
```

Comment:
Skipped tests?

Reply:
iot test was taken from master, few of them is not supported in 2020-09-01-hybrid and hence it was marked for skipped. 
i think the skip description is not very accurate. i will remove these test cases in next PR along with removing old profile test cases.